### PR TITLE
feat: create a custom form for stripe payments

### LINKED ISSUE #334: Make the payments screen for stripe look more authentic
Will make a custom form using editTexts
working on it

### PR DESCRIPTION
**Pull request by nikit19**
fixes #334 
Changes: More padding added to make it comparable with Eventbrite, Changed the accent color from pink to light blue, postal code and country added to the card object
Screenshot:
screenshot_20180718-222851

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt [hunk 1]
```diff
@@ -103,12 +106,75 @@ class AttendeeFragment : Fragment() {
...
+        rootView.cardSelector.adapter = ArrayAdapter(context, android.R.layout.simple_spinner_dropdown_item, cardType)
+        rootView.cardSelector.onItemSelectedListener = object : AdapterView.OnItemSelectedListener {
+            override fun onNothingSelected(p0: AdapterView<*>?) {
+            }
+
+            override fun onItemSelected(p0: AdapterView<*>?, p1: View?, p2: Int, p3: Long) {
+                cardBrand = cardType[p2]
+                rootView.selectCard.text = cardBrand
+            }
+        }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: More than half the logic can go in VM

**Comment by nikit19** [comment]
@iamareebjamal moved the logic to the viewModel. Please review

File: app/src/main/res/layout/fragment_attendee.xml [hunk 2]
```diff
@@ -195,13 +195,154 @@
...
                 android:layout_height="wrap_content"
-                android:visibility="gone" />
+                android:orientation="vertical"
+                android:visibility="gone">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/layout_margin_extra_large"
+                    android:text="Payment Info"
```
Comment (dreadpool2) [inline app/src/main/res/layout/fragment_attendee.xml]: Access from strings.xml

Comment (nikit19) [reply app/src/main/res/layout/fragment_attendee.xml]: PR updated

**Comment by dreadpool2** [comment]
@nikit19 travis is failing

**Pull request merged**